packages/cadl-ranch-specs/http/models/collections-basic/mockapi.ts

```diff
@@ -0,0 +1,24 @@
+import { passOnSuccess, mockapi, json } from "@azure-tools/cadl-ranch-api";
+import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
+
+export const Scenarios: Record<string, ScenarioMockApi> = {};
+
+Scenarios.CollectionPropertiesBasic_sendCollectionModel = passOnSuccess(
+  mockapi.post("/models/collections/basic", (req) => {
+    const expectedBody = { requiredStringList: ["post"], requiredIntList: [100] };
```

Comment:
Is it better we just check whether the field exists instead of a hardcode value?

Reply:
I think we could discuss about the check rules in next LLC scrum meeting